cvat/apps/events/event.py

```diff
@@ -20,6 +20,8 @@ class EventScopes:
         "task": ["create", "update", "delete"],
         "job": ["create", "update", "delete"],
         "organization": ["create", "update", "delete"],
+        "membership": ["create", "update", "delete"],
+        "invitation": ["create", "delete"],
```

Comment:
I don't think it's needed to store the key in the payload for security reasons.
```json
{"key":"nOifLN8CoDO12LjXTE7H4bb4Q9uSE2FiHJHJMS4jdedYO0yBRvOzz5UVGeVpXPU4","created_date":"2024-11-21T17:02:07.674430Z","owner":{"id":1,"username":"admin","first_name":"","last_name":""},"role":"worker","user":{"id":2,"username":"user_1@example.com","first_name":"","last_name":""},"expired":null,"organization_info":{"id":1,"slug":"fdgdf"},"request":{"id":"d4d5358e-81b6-4b7a-82b4-8b7dce5625ce"}}
```

Reply:
key not stored anymore